src/themes/greyVest/SearchEditor.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import { observer } from 'mobx-react'
+import { Flex } from '../../'
+import LinkButton from './LinkButton'
+import TreePauseButton from './TreePauseButton'
+import ToggleFiltersButton from './ToggleFiltersButton'
+
+let SearchEditor = ({ mode, setMode, children, QueryBuilder, FiltersBox }) => {
```

Comment:
I chose `SearchEditor` instead of `SearchFilters` to avoid confusion with `FilterList`, but I'm open to either

Reply:
I like `SearchFilters` better because it's easier to read and learning about `FilterList` vs `SearchEditor` is a relatively O(1) thing. We should bring someone else to review though.